src/Fetch.fs

```diff
@@ -11,6 +11,28 @@ open Browser.Types
 
 [<AutoOpen>]
 module Types =
+    [<StringEnum; RequireQualifiedAccess>]
+    type RequestDestination =
+        | [<CompiledName("")>] Empty
```

Comment:
Does this work? I've never used an empty string 😅

Reply:
I have no idea to be honest 🤣 I hope you would tell me if that was wrong, but the property can indeed be an empty string